src/Microsoft.AspNetCore.Authentication/RemoteAuthenticationOptions.cs

```diff
@@ -56,6 +56,11 @@ public string DisplayName
         /// </summary>
         public bool SaveTokensAsClaims { get; set; }
 
+        /// <summary>
+        /// Gets or sets the lifetime of the correlation cookie used to prevent cross-site scripting attacks.
```

Comment:
If you think `RAT` is clearer, I'm fine :smile: 

I updated the Twitter handler to use the new property. TBH, I'd really like to update it to also use `GenerateCorrelationId`/`ValidateCorrelationId` like the other providers (instead of the custom `RequestToken` and the associated serializer), but I guess it's a bit late for that.